Library/ViewModels/NoShippingPledgeViewModelTests.swift

```diff
@@ -5285,4 +5287,97 @@ final class NoShippingPledgeViewModelTests: TestCase {
       self.showPledgeOverTimeUI.assertValues([false])
     }
   }
+
+  func testPledgeOverTimeConfigData_LoadsAfterBuildPledgeQuery() {
```

Comment:
These tests are pretty bare-bones. I'd love to test that the mock query is actually making a GraphQL request for the right amount, but I don't think we have that capability with our current mock infrastructure.